userspace/libsinsp/container.cpp

```diff
@@ -416,13 +418,14 @@ bool sinsp_container_engine_docker::resolve(sinsp_container_manager* manager, si
 		manager->add_container(container_info, tinfo);
 		manager->notify_new_container(container_info);
 	}
+	return true;
 }
 
-sinsp_docker_response sinsp_container_engine_docker::get_docker(const sinsp_container_manager* manager, const string& url, string &json)
+sinsp_docker_response sinsp_container_engine_docker::get_docker(sinsp_container_manager* manager, const string& url, string &json)
 {
 	const char* response;
-	bool qdres = wh_query_docker(manager->m_inspector->get_wmi_handle(), 
-		(char*)url.c_str(), 
+	bool qdres = wh_query_docker(manager->get_inspector()->get_wmi_handle(),
```

Comment:
Nit: This isn't directly related to your change, but can you init `response` on line 426 to `nullptr`?